### PR TITLE
add testGetShop_success()

### PR DESCRIPTION
**Pull request by KarishmaSinghChauhan**

### CODE REVIEW

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 1]
```diff
@@ -98,6 +98,22 @@ public void testGetTokensForShop_success() throws Exception {
...
+        // Assert.
+        assertEquals("Shop Owner Id ", SHOP_OWNER_ID, newShop.getShopOwnerId());
+        assertEquals("Shop Name ", SHOP_NAME, newShop.getShopName());
+        assertEquals("Shop Address ", SHOP_ADDRESS, newShop.getAddress());
+        assertEquals("Shop Phone Number ", PHONE_NUMBER, newShop.getPhoneNumber());
+        assertEquals("Shop Type ", SHOP_TYPE, newShop.getShopType());
+    }
+
     private UUID addShopToRepository() {
         Shop shop = new Shop(SHOP_OWNER_ID, SHOP_NAME, SHOP_ADDRESS, PHONE_NUMBER, SHOP_TYPE);
```
Comment (sdumka) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:118]: Can we pass these 5 values as params to the  addShopToRepository method. That will make the tests more readable. Right now, we need to look into the logic of addShopToRepository to understand the assert statements and to understand from where are values SHOP_OWNER_ID, SHOP_NAME, etc in assert statements are coming from. If we pass these as params to addShopToRepository method, we'll make each test more independent and more readable.

Comment (KarishmaSinghChauhan) [reply backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:118]: Done.

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 2]
```diff
@@ -98,7 +98,23 @@ public void testGetTokensForShop_success() throws Exception {
...
+
+        // Assert.
+        assertEquals("Shop Owner Id ", SHOP_OWNER_ID, newShop.getShopOwnerId());
+        assertEquals("Shop Name ", SHOP_NAME, newShop.getShopName());
+        assertEquals("Shop Address ", SHOP_ADDRESS, newShop.getAddress());
+        assertEquals("Shop Phone Number ", PHONE_NUMBER, newShop.getPhoneNumber());
+        assertEquals("Shop Type ", SHOP_TYPE, newShop.getShopType());
+    }
+
+    private UUID addShopToRepository(String SHOP_OWNER_ID, String SHOP_NAME, String SHOP_ADDRESS, String PHONE_NUMBER, String SHOP_TYPE) {
```
Comment (sdumka) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:117]: Follow java variable naming convention

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java [hunk 3]
```diff
@@ -98,7 +98,23 @@ public void testGetTokensForShop_success() throws Exception {
...
         Shop shop = new Shop(SHOP_OWNER_ID, SHOP_NAME, SHOP_ADDRESS, PHONE_NUMBER, SHOP_TYPE);
```
Comment (sdumka) [inline backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:118]: Why use these when the values are being passed as params to this method?

Comment (sdumka) [reply backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java:118]: These will change when the variable names in the params of this method changes

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java (hunk 2 above)
Comment (KarishmaSinghChauhan) [reply backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java]: Done.

File: backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java (hunk 3 above)
Comment (KarishmaSinghChauhan) [reply backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java]: Changed.

**Pull request merged**